File: beginner/javascript/2031.js
var input = require("fs").readFileSync("stdin", "utf8");
var lines = input.split("\n");

var n = parseInt(lines.shift());

for(var i = 0; i < n; i++){
    var j1 = lines.shift();
    var j2 = lines.shift();

    if (j1 == "ataque" &&j2 == "ataque")
        console.log("Aniquilacao mutua")
    else if (j1 == "pedra" && j2 == "pedra")
        console.log("Sem ganhador")
    else if (j1 == "papel" && j2 == "papel")
        console.log("Ambos venceram")
    else if (j1 == "ataque" && j2 == "pedra")
        console.log("Jogador 1 venceu")
    else if (j1 == "pedra" && j2 == "ataque")
        console.log("Jogador 2 venceu")
    else if (j1 == "pedra" && j2 == "papel")
        console.log("Jogador 1 venceu")
    else if (j1 == "papel" && j2 == "pedra")
        console.log("Jogador 2 venceu")
    else if (j1 == "ataque" && j2 == "papel")
        console.log("Jogador 1 venceu")
    else if (j1 == "papel" && j2 == "ataque")
        console.log("Jogador 2 venceu")

}

File: beginner/python/2031.py
n = int(input())

for _ in range(n):
    j1 = input()
    j2 = input()

    if j1 == j2 == "ataque":
        print("Aniquilacao mutua")
    elif j1 == j2 == "pedra":
        print("Sem ganhador")
    elif j1 == j2 == "papel":
        print("Ambos venceram")
    elif j1 == "ataque" and j2 == "pedra":
        print("Jogador 1 venceu")
    elif j1 == "pedra" and j2 == "ataque":
        print("Jogador 2 venceu")
    elif j1 == "pedra" and j2 == "papel":
        print("Jogador 1 venceu")
    elif j1 == "papel" and j2 == "pedra":
        print("Jogador 2 venceu")
    elif j1 == "ataque" and j2 == "papel":
        print("Jogador 1 venceu")
    elif j1 == "papel" and j2 == "ataque":
        print("Jogador 2 venceu")
